system/jlib/jfile.cpp

```diff
@@ -4561,6 +4562,16 @@ StringBuffer & RemoteFilename::getPath(StringBuffer & name) const
     else
         return getRemotePath(name);
 }
+StringBuffer  & RemoteFilename::getPathInStoragePlane(StringBuffer & name) const
+{
+    if (isLocal())
+        getLocalPath(name);
+    else
+        getRemotePath(name);
+    if (mountPathLength)
```

Comment:
local paths are going to have mountPathLength set to 0.   It will only be set to a value for a remote mount. It would be clearer to rename this to mountPathLengthForRemoteMounts?  Any suggestions on this for shorter more descriptive name? @ghalliday @jakesmith

Reply:
well, I think these changes to RemoteFilename should be removed, see main review comment(s) here : https://github.com/hpcc-systems/HPCC-Platform/pull/16317#pullrequestreview-1039425771 (and the one it points to)